lib/linguist/languages.yml

```diff
@@ -2053,6 +2080,37 @@ Idris:
   ace_mode: text
   tm_scope: source.idris
   language_id: 165
+Ignore List:
+  type: data
+  group: INI
+  aliases:
+  - ignore
+  - gitignore
+  - git-ignore
+  extensions:
+  - ".gitignore"
+  filenames:
+  - ".atomignore"
+  - ".babelignore"
+  - ".bzrignore"
+  - ".coffeelintignore"
+  - ".cvsignore"
+  - ".dockerignore"
+  - ".eslintignore"
+  - ".gitignore"
+  - ".nodemonignore"
+  - ".npmignore"
+  - ".prettierignore"
+  - ".stylelintignore"
+  - ".vscodeignore"
+  - gitignore
```

Comment:
Are you sure there are files on GitHub matching that filename? [I tried to search for them](https://github.com/search?p=1&q=filename%3Agitignore+NOT+nothack&type=Code&utf8=%E2%9C%93), but I don't think there are any because, if there were, I think GitHub search would return them first, before other "close matches".

Reply:
Not 100% sure, no, but is that the way how GitHub's searches are ordered? I couldn't figure out how to search for a filename *verbatim* (without GitHub broadening it to include whatever 😢).

Reply:
@lildude Any idea?

Reply:
Nope. 😓 Although the filename itself makes sense if you need to track a `gitignore` file without having it interpreted by Git itself...

Reply:
Ah, I see that now - sorry for the noise.

Reply:
> I couldn't figure out how to search for a filename verbatim
> @lildude Any idea?

Yup: [`filename:gitignore`](https://github.com/search?q=filename%3Agitignore)

Reply:
Should we remove that filename then?

Reply:
Didn't even see this comment until just now, sorry. 😓